project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject quil "3.1.1-SNAPSHOT"
+(defproject quil "4.0.0-SNAPSHOT-1"
```

Comment:
I created `processing4` branch so that master is still on 3 in case we need to fix some issues there. Can you change your pull request to merge into the new branch?